src/main/java/mate/academy/service/MovieService.java

```diff
@@ -9,4 +9,8 @@
     Movie get(Long id);
 
     List<Movie> getAll();
+
+    Boolean update(Movie movie);
+
+    Boolean delete(Long id);
```

Comment:
Consider changing the return type of the `delete` method from `Boolean` to `boolean` to avoid unnecessary boxing and potential nullability issues.